bgcval2/functions/TotalIntPP.py

```diff
@@ -84,6 +86,10 @@ def MA_TotalIntPP(nc, keys, **kwargs):
     # This will only work for NEMO models:
     thkfn = nc.filename.replace('diag_T', 'grid_T').replace('ptrc_T', 'grid_T')
     print('MA_TotalIntPP: opening:', thkfn)
+    if not os.path.exists(thkfn):
+       print('MA_TotalIntPP: ERROR: does not exist:', thkfn)  
+       return np.ma.masked
```

Comment:
np.ma.masked what? That's just a generic Numpy function

Reply:
It's a masked value, so that the arrays all line up corectly, even if there's no data here.

Reply:
that's a `<class 'numpy.ma.core.MaskedConstant'>` which is a stuffy class (that, in effect is just a `--` with a value of , all you need is the value that gets masked (1e20 or -999 or whatever). Note that the fillvalue of np.ma.masked is not always what one needs be, it's 999999 so that might screw up things https://numpy.org/doc/stable/reference/maskedarray.baseclass.html